### DISCUSSION TITLE
I keep getting deployment error

### DISCUSSION THREAD

**Discussion by Temiswift** [Cyfrin/foundry-full-course-cu#2871] [Q&A]
After deploying my contract on the ZKsync sepolia testnet, It brought out this error, Should i be worried? Although it's looking like the contract might have been deployed successfully
![Deployment error](https://github.com/user-attachments/assets/c3ccb3ed-3b07-4575-ac87-e1c4188139aa)

**Comment by EngrPips** [comment]
Hello @Temiswift, Did you figure this out already?

**Reply by Temiswift** [reply]
Yes, I think so cause on zksync sepolia explorer it's showing that i've already deployed my smart contract and i can see all transactions on it

Thanks!!

**Reply by EngrPips** [reply]
Awesome. Keep crushing it.

**Reply by Jozewin** [reply]
> Hello @Temiswift, Did you figure this out already?

The contract has been deployed but were you you able to verify the contract? coz im too getting this same error and stuck here for 2 days ;-;

**Reply by EngrPips** [reply]
Hello @Jozewin, If the contract gets deployed successfully, I wouldn't bother too much with verification. Working with `ZKSync` isn't completely smooth as of now.

**Reply by Jozewin** [reply]
Thanks brother @EngrPips I don't know what I did but it got verified and deployed. I just removed every comment from my code and it got verified. As you said I'll just focus on the deployed code rather than verify🫡

**Reply by EngrPips** [reply]
> Thanks brother @EngrPips I don't know what I did but it got verified and deployed. I just removed every comment from my code and it got verified. As you said I'll just focus on the deployed code rather than verify🫡

Intersting.

**Comment by davidafolabi20** [comment]
This may still be relevant, I had the challenge as well and was just about to give up till I saw the next video which was about a fix for the zksync plugin. It was creating a challenge that was different from this but the solution works as well because I can now deploy and verify successfuly.

Move your simplestorage.sol contract into folder : )

**Reply by EngrPips** [reply]
Awesome, Great heads-up. Keep crushing it.
